README.md

```diff
@@ -201,6 +201,13 @@ var railsImagePath = "<%= helpers.image_path('rails.png') %>";
 
 This is enabled by the `rails-erb-loader` loader rule in `config/webpack/shared.js`.
 
+## Ready for Elm
```

Comment:
Lets do this for now and we can revisit this later. I have some plans for README 👍